src/Root.sol

```diff
@@ -74,6 +74,7 @@ contract Root is Auth {
 
     /// @notice Cancel a pending scheduled rely
     function cancelRely(address target) external auth {
+        require(schedule[target] != 0, "Root/target-not-scheduled");
```

Comment:
shouldn't it be `address(0`)

Reply:
The `schedule[target]` value is the timestamp when it can be relied

Reply:
aaaaah, nvm 🙈